src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -16,12 +17,13 @@
      */
     public synchronized void setCurrentDriver(Job2dDriver driver) {
         currentDriver = driver;
+        UsageMonitorManager.setDriver(currentDriver);
     }
 
     /**
      * @return Current driver.
      */
     public synchronized Job2dDriver getCurrentDriver() {
-        return currentDriver;
+        return UsageMonitorManager.getDriver();
```

Comment:
I would prefer not to change DriverManager at the moment. Usage monitoring is to a specific function and should not be forced into the current design. It is better to make it a little less intrusive for a moment (even sacrificing its effectiveness)

Reply:
The dedicated monitored driver has been added to tests instead of changing the DriverManager.